theano/sandbox/tests/test_multinomial_wo_replacement.py

```diff
@@ -43,13 +46,13 @@ def test_select_distinct(self):
 
     def test_fail_select_alot(self):
         """
-        Tests that MultinomialWOReplacementFromUniform fails when asked to sample more
+        Tests that ChoiceFromUniform fails when asked to sample more
         elements than the actual number of elements
         """
         p = tensor.fmatrix()
         u = tensor.fvector()
         n = tensor.iscalar()
-        m = multinomial.MultinomialWOReplacementFromUniform('auto')(p, u, n)
+        m = multinomial.ChoiceFromUniform('auto')(p, u, n)
```

Comment:
Similar rename is probably needed in the file theano/sandbox/rng_mrg.py. To make sure we didn't miss such changes, can you run Theano tests while commenting the line "MultinomialWOReplacementFromUniform = ChoiceFromUniform" and the equivalent GPU one? If there is rename forgotten, then this will causes errors.